src/iOS.Core/Views/FormEntryTableViewCell.cs

```diff
@@ -112,7 +112,7 @@ public class FormEntryTableViewCell : ExtendedUITableViewCell, ISelectable
                 ContentView.Add(TextField);
                 ContentView.AddConstraints(new NSLayoutConstraint[] {
                     NSLayoutConstraint.Create(TextField, NSLayoutAttribute.Leading, NSLayoutRelation.Equal, ContentView, NSLayoutAttribute.Leading, 1f, leadingConstant),
-                    NSLayoutConstraint.Create(ContentView, NSLayoutAttribute.Trailing, NSLayoutRelation.Equal, TextField, NSLayoutAttribute.Trailing, 1f, useButton ? 55f : 15f),
+                    NSLayoutConstraint.Create(ContentView, NSLayoutAttribute.Trailing, NSLayoutRelation.Equal, TextField, NSLayoutAttribute.Trailing, 1f, useTwoButtons ? 95f : useButton ? 55f : 15f),
```

Comment:
I'm not fond of using nested ternary operator, I'd just use a local function with a switch and return the proper size depending on the case